### ISSUE TITLE
[leaderboard]

### ISSUE THREAD

**Issue by mgranz** [Jingkang50/OpenOOD#270]
Dear OpenOOD team, 

first we would like to thank you for your work and dedication. OpenOOD really helped the OOD community to correctly verify results and move forward. We would like to add our postprocessor to your leaderboard. Here are our results on four of your benchmarks:
 
## CIFAR10
- **`Training`**: `CrossEntropy`.
- **`Postprocessor`**: `WeiPer+KLD`
- **`Near-OOD AUROC`**: `90.54`
- **`Far-OOD AUROC`**: `93.12` 
- **`ID Accuracy`**: `95.06`
- **`Outlier Data`**: `No`
- **`Model Arch.`**: `ResNet18`.
- **`Additional Description`**: `100 epochs` (OpenOOD checkpoints)

## CIFAR100
- **`Training`**: `CrossEntropy`.
- **`Postprocessor`**: `WeiPer+KLD`
- **`Near-OOD AUROC`**: `81.37`
- **`Far-OOD AUROC`**: `79.01` 
- **`ID Accuracy`**: `77.25`
- **`Outlier Data`**: `No`
- **`Model Arch.`**: `ResNet18`.
- **`Additional Description`**: `100 epochs` (OpenOOD checkpoints)

## ImageNet
- **`Training`**: `CrossEntropy`.
- **`Postprocessor`**: `WeiPer+KLD`
- **`Near-OOD AUROC`**: `80.05`
- **`Far-OOD AUROC`**: `95.54` 
- **`ID Accuracy`**: `76.18`
- **`Outlier Data`**: `No`
- **`Model Arch.`**: `ResNet50`.
- **`Additional Description`**: `torchvision pretrained`

## ImageNet
- **`Training`**: `CrossEntropy`.
- **`Postprocessor`**: `WeiPer+KLD`
- **`Near-OOD AUROC`**: `75.00`
- **`Far-OOD AUROC`**: `90.32` 
- **`ID Accuracy`**: `81.14`
- **`Outlier Data`**: `No`
- **`Model Arch.`**: `ViT-B/16`.
- **`Additional Description`**: `torchvision pretrained`

You can find our paper and code here:

[Github](https://github.com/mgranz/weiper)
[Openreview](https://openreview.net/forum?id=8HeUvbImKT&referrer=%5Bthe%20profile%20of%20Maximilian%20Granz%5D(%2Fprofile%3Fid%3D~Maximilian_Granz1))

Thanks again for your work!

Best regards,

Maximilian Granz

**Comment by zjysteven** [comment]
Congrats on the NeurIPS acceptance! WeiPer has now been included in the leaderboard.
